pandas/conftest.py

```diff
@@ -28,28 +28,13 @@
 
 
 def pytest_addoption(parser):
-    parser.addoption("--skip-slow", action="store_true",
```

Comment:
i wouldn't remove these, these are useful from the command line

Reply:
Any reason why to use `pytest --skip-slow` instead of `pytest -m "no slow"`? Not a big deal  to restore them, but I personally think that they are more complexity than value.

Reply:
yes this is a pretty common idiom. I don't see value in removing them.

Reply:
@datapythonista can you restore these